Sources/SourceKitLSP/Swift/SwiftLanguageService.swift

```diff
@@ -815,15 +831,31 @@ extension SwiftLanguageService {
       additionalParameters: additionalCursorInfoParameters
     )
 
-    return cursorInfoResponse.refactorActions.compactMap {
-      do {
-        let lspCommand = try $0.asCommand()
-        return CodeAction(title: $0.title, kind: .refactor, command: lspCommand)
-      } catch {
-        logger.log("Failed to convert SwiftCommand to Command type: \(error.forLogging)")
-        return nil
+    var canInlineMacro = false
+
+    let showMacroExpansionsIsEnabled = await self.sourceKitLSPServer?.options.experimentalFeatures.contains(
+      .showMacroExpansions
+    )
```

Comment:
Formatting nitpick again 😉 Also, I would just nil-coalesce here so we don’t need to deal with `showMacroExpansionsIsEnabled == nil` below

```suggestion
    let showMacroExpansionsIsEnabled = await self.sourceKitLSPServer?.options.experimentalFeatures
      .contains(.showMacroExpansions) ?? false
```